subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/PmdPlugin.java

```diff
@@ -40,7 +40,7 @@
  */
 public class PmdPlugin extends AbstractCodeQualityPlugin<Pmd> {
 
-    public static final String DEFAULT_PMD_VERSION = "6.21.0";
+    public static final String DEFAULT_PMD_VERSION = "6.23.0";
```

Comment:
We should also bump the version of the documentation links in `Pmd` and `PmdExtension`

Reply:
Thanks. I wasn't aware of the docs. Fixed that.